tests/translator/input/error_function_invalid_autopublishalias.yaml

```diff
@@ -0,0 +1,115 @@
+Parameters:
+  MyAlias:
+    Type: String
+    Default: ""
+Resources:
+  InvalidAutoPublishAliasFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.lambda_handler
+      CodeUri: s3://bucket/object
+      Runtime: python3.7
+      AutoPublishAlias:
+        Ref: MyAlias
+      Events:
+        Get:
+          Type: Api
+          Properties:
+            Path: "/path"
+            Method: GET
+# ---
+# Parameters:
```

Comment:
Why do we have all of this if it is all commented out?

Reply:
Removed